Index: telegram-bot-command/src/main/java/pro/sky/telegrambot/service/TelegramBotUpdatesListener.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/telegram-bot-command/src/main/java/pro/sky/telegrambot/service/TelegramBotUpdatesListener.java b/telegram-bot-command/src/main/java/pro/sky/telegrambot/service/TelegramBotUpdatesListener.java
--- a/telegram-bot-command/src/main/java/pro/sky/telegrambot/service/TelegramBotUpdatesListener.java	
+++ b/telegram-bot-command/src/main/java/pro/sky/telegrambot/service/TelegramBotUpdatesListener.java	
@@ -16,11 +16,17 @@
 import java.util.List;
 
 @Service
-public class TelegramBotUpdatesListener implements UpdatesListener {
+public class  TelegramBotUpdatesListener implements UpdatesListener {
 
     private Logger logger = LoggerFactory.getLogger(TelegramBotUpdatesListener.class);
     private final ReplyKeyboards replyKeyboards = new ReplyKeyboards();
 
+    private final ShelterService shelterService;
+
+    public TelegramBotUpdatesListener(ShelterService shelterService) {
+        this.shelterService = shelterService;
+    }
+
     @Autowired
     private TelegramBot telegramBot;
 
@@ -39,6 +45,10 @@
     }
 
     private void messageProcessing(Update update){
+
+        try {
+
+
         long chatId = update.message().chat().id();
         switch (update.message().text()) {
             case "/start":
@@ -64,12 +74,22 @@
                 SendMessage messageFour = new SendMessage(chatId, Constants.WELCOME_MESSAGE_FOUR);
                 telegramBot.execute(messageFour);
                 break;
+            case "/init":
+                shelterService.updateInfo();
+                break;
+            case "/hello":
+                SendMessage message = new SendMessage(chatId, shelterService.greetings());
+                telegramBot.execute(message);
+                break;
 
                 // СЮДА НАПИСАТЬ ДЕЙСТВИЯ НА ОСТАВШИЕСЯ КНОПКИ
 
             default:
-                SendMessage message = new SendMessage(chatId, Constants.SORRY_MESSAGE);
-                telegramBot.execute(message);
+                SendMessage messages = new SendMessage(chatId, Constants.SORRY_MESSAGE);
+                telegramBot.execute(messages);
+        }
+    }catch (Exception e){
+
         }
     }
 
Index: telegram-bot-command/src/main/java/pro/sky/telegrambot/model/Shelter.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/telegram-bot-command/src/main/java/pro/sky/telegrambot/model/Shelter.java b/telegram-bot-command/src/main/java/pro/sky/telegrambot/model/Shelter.java
--- a/telegram-bot-command/src/main/java/pro/sky/telegrambot/model/Shelter.java	
+++ b/telegram-bot-command/src/main/java/pro/sky/telegrambot/model/Shelter.java	
@@ -4,27 +4,30 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
+import pro.sky.telegrambot.TelegramBotApplication;
 
 import java.io.*;
+import java.net.URISyntaxException;
 import java.nio.file.Files;
 import java.nio.file.Paths;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
 import java.util.stream.Collectors;
 
 @Component
 public class Shelter {
-    @Value("${greetingsFileName}")
+    @Value("${greetings.FileName}")
     private String greetingsFileName;
-    @Value("${descriptionFileName}")
+    @Value("${description.FileName}")
     private String descriptionFileName;
-    @Value("${scheduleFileName}")
+    @Value("${schedule.FileName}")
     private String scheduleAndAddressFileName;
-    @Value("${documentsForAdoptionFileName}")
+    @Value("${documentsForAdoption.FileName}")
     private String documentsForAdoptionFileName;
-    @Value("${safetyPrecuationsFileName")
+    @Value("${safetyPrecuations.FileName")
     private String safetyPrecuationsFileName;
-    @Value("${declineReasonsFileName}")
+    @Value("${declineReasons.FileName}")
     private String declineReasonsFileName;
 
 
@@ -42,7 +45,7 @@
 
     public Shelter() {
         System.out.println("Вызываю конструктор приюта");
-        updateInfoAboutShelter();
+        //updateInfoAboutShelter();
     }
 
     public void updateInfoAboutShelter() {
@@ -56,11 +59,25 @@
     }
 
     private List<String> readStringsFromFile(String fileName) {
+        if(fileName==null){
+            return null;
+        }
         List<String> strings = new ArrayList<>();
+        try {
+            strings = Files.readAllLines(Paths.get(Objects.requireNonNull(TelegramBotApplication.class.getResource(fileName)).toURI()));
+        } catch (IOException e) {
+
+        } catch (URISyntaxException e) {
+
+        }
+        //System.out.println(strings);
+        return strings;
+
+        /*
         try (
                 BufferedReader br = Files.newBufferedReader(Paths.get(fileName))
               /* FileReader fr = new FileReader(fileName);
-                BufferedReader reader = new BufferedReader(fr)*/) {
+                BufferedReader reader = new BufferedReader(fr)) {
             while (br.ready()) {
                 strings.add(br.readLine());
             }
@@ -75,6 +92,7 @@
         } finally {
             return strings;
         }
+        */
     }
 
 
Index: telegram-bot-command/src/main/java/pro/sky/telegrambot/service/ShelterService.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/telegram-bot-command/src/main/java/pro/sky/telegrambot/service/ShelterService.java b/telegram-bot-command/src/main/java/pro/sky/telegrambot/service/ShelterService.java
--- a/telegram-bot-command/src/main/java/pro/sky/telegrambot/service/ShelterService.java	
+++ b/telegram-bot-command/src/main/java/pro/sky/telegrambot/service/ShelterService.java	
@@ -7,7 +7,6 @@
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Paths;
-import java.util.List;
 
 @Service
 public class ShelterService {
@@ -26,6 +25,7 @@
         return shelter.getAbout();
     }
 
+
     public String getSheduleAndAdress() {
         return shelter.getScheduleAndAdress();
     }
@@ -47,13 +47,4 @@
         shelter.updateInfoAboutShelter();
     }
 
-    public String hello() {
-        try (BufferedReader br = Files.newBufferedReader(     Paths.get("C:\\Users\\mishutkin.va\\IdeaProjects\\commandProject\\telegram-bot-command\\src\\main\\resources\\files\\hello.txt"))){
-            return br.readLine();
-        }catch (IOException e){
-
-        }
-     return null;
-
-    }
 }
Index: telegram-bot-command/src/main/java/pro/sky/telegrambot/controller/TelegramBotController.java
===================================================================
diff --git a/telegram-bot-command/src/main/java/pro/sky/telegrambot/controller/TelegramBotController.java b/telegram-bot-command/src/main/java/pro/sky/telegrambot/controller/TelegramBotController.java
deleted file mode 100644
--- a/telegram-bot-command/src/main/java/pro/sky/telegrambot/controller/TelegramBotController.java	
+++ /dev/null	
@@ -1,4 +0,0 @@
-package pro.sky.telegrambot.controller;
-
-public class TelegramBotController {
-}
Index: telegram-bot-command/target/classes/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/telegram-bot-command/target/classes/application.properties b/telegram-bot-command/target/classes/application.properties
--- a/telegram-bot-command/target/classes/application.properties	
+++ b/telegram-bot-command/target/classes/application.properties	
@@ -1,12 +1,14 @@
 spring.datasource.url=jdbc:postgresql://localhost:5432/shelterTelegramBot
 spring.datasource.username=command
 spring.datasource.password=1234567890
-telegram.bot.token=58980
+telegram.bot.token=5640743318:AAEg_UU1aDGgLax5ZW1yadaSf-sTq2mxKTM
 
-greetingsFileName= C:\\Users\\mishutkin.va\\IdeaProjects\\commandProject\\telegram-bot-command\\src\\main\\resources\\files\\greetings.txt
-descriptionFileName=C:\\Users\\mishutkin.va\\IdeaProjects\\commandProject\\telegram-bot-command\\src\\main\\resources\\files\\shelterDescription.txt
-scheduleFileName=C:\\Users\\mishutkin.va\\IdeaProjects\\commandProject\\telegram-bot-command\\src\\main\\resources\\files\\scheduleAndAddress.txt
-documentsForAdoptionFileName=C:\\Users\\mishutkin.va\\IdeaProjects\\commandProject\\telegram-bot-command\\src\\main\\resources\\files\\documentsForAdoption.txt
-declineReasonsFileName=C:\\Users\\mishutkin.va\\IdeaProjects\\commandProject\\telegram-bot-command\\src\\main\\resources\\files\\declineReasons.txt
+greetings.FileName=/greetings.txt
+description.FileName=/shelterDescription.txt
+schedule.FileName=/scheduleAndAddress.txt
+documentsForAdoption.FileName=/documentsForAdoption.txt
+declineReasons.FileName=/declineReasons.txt
+safetyPrecuations.FileName=/safetyPrecuations.txt
+
 
 
Index: telegram-bot-command/src/main/java/pro/sky/telegrambot/TelegramBotApplication.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/telegram-bot-command/src/main/java/pro/sky/telegrambot/TelegramBotApplication.java b/telegram-bot-command/src/main/java/pro/sky/telegrambot/TelegramBotApplication.java
--- a/telegram-bot-command/src/main/java/pro/sky/telegrambot/TelegramBotApplication.java	
+++ b/telegram-bot-command/src/main/java/pro/sky/telegrambot/TelegramBotApplication.java	
@@ -2,7 +2,6 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
 import org.springframework.scheduling.annotation.EnableScheduling;
 
 @SpringBootApplication
Index: telegram-bot-command/src/main/resources/safetyPrecuations.txt
===================================================================
diff --git a/telegram-bot-command/src/main/resources/safetyPrecuations.txt b/telegram-bot-command/src/main/resources/safetyPrecuations.txt
new file mode 100644
--- /dev/null	
+++ b/telegram-bot-command/src/main/resources/safetyPrecuations.txt	
@@ -0,0 +1,2 @@
+Не балуйтесь!
+вы слишком много балуетесь
\ No newline at end of file
diff --git a/telegram-bot-command/src/main/resources/files/shelterDescription.txt b/telegram-bot-command/src/main/resources/shelterDescription.txt
rename from telegram-bot-command/src/main/resources/files/shelterDescription.txt
rename to telegram-bot-command/src/main/resources/shelterDescription.txt
diff --git a/telegram-bot-command/src/main/resources/files/scheduleAndAddress.txt b/telegram-bot-command/src/main/resources/scheduleAndAddress.txt
rename from telegram-bot-command/src/main/resources/files/scheduleAndAddress.txt
rename to telegram-bot-command/src/main/resources/scheduleAndAddress.txt
diff --git a/telegram-bot-command/src/main/resources/files/declineReasons.txt b/telegram-bot-command/src/main/resources/declineReasons.txt
rename from telegram-bot-command/src/main/resources/files/declineReasons.txt
rename to telegram-bot-command/src/main/resources/declineReasons.txt
diff --git a/telegram-bot-command/src/main/resources/files/greetings.txt b/telegram-bot-command/src/main/resources/greetings.txt
rename from telegram-bot-command/src/main/resources/files/greetings.txt
rename to telegram-bot-command/src/main/resources/greetings.txt
diff --git a/telegram-bot-command/src/main/resources/files/documentsForAdoption.txt b/telegram-bot-command/src/main/resources/documentsForAdoption.txt
rename from telegram-bot-command/src/main/resources/files/documentsForAdoption.txt
rename to telegram-bot-command/src/main/resources/documentsForAdoption.txt
