Index: telegram-bot-command/src/main/resources/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/telegram-bot-command/src/main/resources/application.properties b/telegram-bot-command/src/main/resources/application.properties
--- a/telegram-bot-command/src/main/resources/application.properties	
+++ b/telegram-bot-command/src/main/resources/application.properties	
@@ -1,6 +1,6 @@
 spring.datasource.url=jdbc:postgresql://localhost:5432/shelterTelegramBot
-spring.datasource.username=command
-spring.datasource.password=1234567890
+spring.datasource.username=myUser
+spring.datasource.password=1qaz2wsx
 telegram.bot.token=5640743318:AAEg_UU1aDGgLax5ZW1yadaSf-sTq2mxKTM
 
 greetings.FileName=/greetings.txt
Index: telegram-bot-command/src/main/java/pro/sky/telegrambot/entitydatabase/Person.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/telegram-bot-command/src/main/java/pro/sky/telegrambot/entitydatabase/Person.java b/telegram-bot-command/src/main/java/pro/sky/telegrambot/entitydatabase/Person.java
--- a/telegram-bot-command/src/main/java/pro/sky/telegrambot/entitydatabase/Person.java	
+++ b/telegram-bot-command/src/main/java/pro/sky/telegrambot/entitydatabase/Person.java	
@@ -12,8 +12,15 @@
     private Long chatId;
     private String numberPhone;
     private String fullName;
+
+    //nickname
     private Boolean status;
 
+
+    //начало испытательного срока
+    // конец испытательного срока
+    //статус усыновления
+
     public Person(Long chatId, String numberPhone, String fullName, Boolean status) {
         this.chatId = chatId;
         this.numberPhone = numberPhone;
Index: telegram-bot-command/src/main/java/pro/sky/telegrambot/entitydatabase/Report.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/telegram-bot-command/src/main/java/pro/sky/telegrambot/entitydatabase/Report.java b/telegram-bot-command/src/main/java/pro/sky/telegrambot/entitydatabase/Report.java
--- a/telegram-bot-command/src/main/java/pro/sky/telegrambot/entitydatabase/Report.java	
+++ b/telegram-bot-command/src/main/java/pro/sky/telegrambot/entitydatabase/Report.java	
@@ -11,7 +11,7 @@
     private Long id;
     private Long chatId;
     private String message;
-
+    //
     public Long getId() {
         return id;
     }
Index: telegram-bot-command/target/classes/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/telegram-bot-command/target/classes/application.properties b/telegram-bot-command/target/classes/application.properties
--- a/telegram-bot-command/target/classes/application.properties	
+++ b/telegram-bot-command/target/classes/application.properties	
@@ -1,6 +1,6 @@
 spring.datasource.url=jdbc:postgresql://localhost:5432/shelterTelegramBot
-spring.datasource.username=command
-spring.datasource.password=1234567890
+spring.datasource.username=myUser
+spring.datasource.password=1qaz2wsx
 telegram.bot.token=5640743318:AAEg_UU1aDGgLax5ZW1yadaSf-sTq2mxKTM
 
 greetings.FileName=/greetings.txt
